setup.py

```diff
@@ -111,7 +109,7 @@ def read_requirements(extra=None):
     # for example:
     # $ pip install -e .[dev,test]
     extras_require={'doc': read_requirements('doc')},
-    setup_requires=['setuptools_scm<6.0'],  # setuptools_scm>=6.0 doesn't support Python 3.5
+    setup_requires=['setuptools_scm'],
```

Comment:
Why not pinning the version?

Reply:
I'd think we want to use the newest one we can find during install and if it crashes, investigate and maybe pin to a lower version then

Reply:
setuptools_scm removed support for setup.py sometimes after 6.0